tests/package-title-dir-rtl-001/OPS/package.opf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+    <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+        <dc:coverage>Internationalization</dc:coverage>
+        <dc:creator>Ivan Herman</dc:creator>
+        <dc:date>2021-09-22</dc:date>
+        <dc:description>The 'dc:title' element contains a text whose proper rendering requires bidi control. The element's 'dir' attribute is set to 'rtl'; the title should display correctly.</dc:description>
```

Comment:
Nit: instead of "correctly," say ... "from right to left," I guess?

Reply:
Done it in https://github.com/w3c/epub-tests/pull/31/commits/dc9f0d396bd4d814812b3d707225bef4f0f59357